vocabularies/voc4cat/0000111.ttl

```diff
@@ -5,6 +5,9 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0000111>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0000111"^^xsd:token ;
+    skos:altLabel
+        "catalysis duration"@en ,
+        "catalysis reaction time"@en ;
```

Comment:
Both altLabel seem to have another meaning then "experiment time". They may be related but they are not synonyms. Please remove.